### ISSUE TITLE
508-defect-3 [COGNITION]: COVID Vaccine Trials - Consider breaking Step 1 into three distinct pages

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#18248]
# [508-defect-3](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-3)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** _Trevor_

## User Story or Problem Statement

As a screen reader user, I had some trouble filling out the form because it was very long, with all of the questions on one page.

## Details

While filling out the form, I had some trouble with the length of the page. There were no headings to break up the sections, so my mental model was "one long form". I think we could consider breaking the form up into three pages, and use the Bitter text styled as headings for the step labels. Screenshots attached below of headings that could be used to label steps.

I previously wrote #18246 to consider making those Bitter text blocks into H3s. This ticket (if the team moves forward) will replace that one for the best course of action.

## Acceptance Criteria

- [ ] 0 axe-core violations on each page
- [ ] Headings are properly announced by screen readers

## Environment

* https://staging.va.gov/coronavirus-research/volunteer/sign-up

## WCAG or Vendor Guidance (optional)

* [Info and Relationships: Understanding SC 1.3.1](http://www.w3.org/TR/UNDERSTANDING-WCAG20/content-structure-separation-programmatic.html)

## Screenshots or Trace Logs

![Screen Shot 2021-01-07 at 12 12 01 PM](https://user-images.githubusercontent.com/934879/103929468-fcdb6600-50e2-11eb-954b-4f758805171d.png)

---
![Screen Shot 2021-01-07 at 12 12 08 PM](https://user-images.githubusercontent.com/934879/103929489-02d14700-50e3-11eb-85d6-4d7e81610e59.png)

---
![Screen Shot 2021-01-07 at 12 12 13 PM](https://user-images.githubusercontent.com/934879/103929503-06fd6480-50e3-11eb-9051-0370f63d3d64.png)

**Comment by stephenBarrs** [comment]
The decision to use a single page form was based on UX studies done with Veterans who preferred the single page form.  #18246 has been implemented to add `h2` sub-headings to increase navigation ability by screen readers.
